neo4j/neo4j_test.go

```
@@ -0,0 +1,211 @@
+	// Initialize Neo4j store with test container credentials
+	store := New(Config{
+		Reset:    true,
+		URI:      uri,
+		Username: "neo4j",
+		Password: "pass#w*#d",
+	})
+
+	testStore = store
+
+	defer testStore.Close()
+	defer neo4jContainer.Terminate(ctx)
```

golangci-lint (1.62.2)

Error return value of `neo4jContainer.Terminate` is not checked

(errcheck)